Sources/Generator/UIGenerator.swift

```diff
@@ -105,7 +110,43 @@ public class UIGenerator: Generator {
                     }
                     try root.children.forEach { try generate(element: $0, superName: "target", containedIn: root) }
                     tempCounter = 1
-                    root.children.forEach { generateConstraints(element: $0, superName: "target") }
+                    root.children.forEach { generateConstraints(element: $0, superName: "target", forUpdate: false) }
+                    if configuration.isLiveEnabled {
+                        l("#endif")
+                    }
+                }
+                l()
+                l("func updateReactantUI()") {
+                    tempCounter = 1
+                    let guardNames = ["target"] + root.children.generatedNames(tempCounter: &tempCounter)
+                    if !guardNames.isEmpty {
```

Comment:
Does this if make sense since there will always me at least `target`?

Reply:
Nice catch.